### DISCUSSION TITLE
birdcall competition

### DISCUSSION THREAD

**Discussion by radekosmulski** [earthspecies/project#6] [General]
Over the weekend I created a new [starter pack](https://www.kaggle.com/c/birdsong-recognition/discussion/166577) which performed more poorly than I have hoped for. Despite this not working as good as I have hoped, it still created a good opportunity to communicate with the community and share some of my thoughts.

Other than that, I am looking into conveniently annotating recordings to train a detection model and see how it performs with a cleaner train set.

**Comment by bs** [comment]
BTW, Holger was at the conference this weekend! @radekosmulski, at what point do you suggest we reach out to the organizing team?

**Reply by radekosmulski** [reply]
Cool that he was at the conference 🙂I think we can meet with them whenever would be most convenient. Frankly speaking, I was hoping I would have some super wonderful insights I could share with the community but so far mostly I have been coming up empty. This week I am planning to work on using models to clean up data and train on the cleaned up data and the novel way for data annotation I came up with - I think both (especially the latter) if they bring some results would make for very interesting discussion points.

**Comment by kzacarian** [comment]
Speaking of birdcalls, I saw this post from [How do I scale up acoustic surveys with Audiomoths and automated processing](https://www.wildlabs.net/resources/events/tech-tutors-recording-how-do-i-scale-acoustic-surveys-audiomoths) featuring Tessa Rhinehart, of the Kitzes Lab at the University of Pittsburgh.  She "develops methods to survey animals on landscape scales. As a Research Programmer, I create machine learning models to automatically classify animal vocalizations... Additionally, between my lab and our collaborators, I've helped to manage over 2,000 AudioMoth autonomous recorders and the data that they generate. You can read more about my lab's work in this recent article written for [Birding magazine](https://www.aba.org/wp-content/uploads/2020/04/Birding_20-2_11-Birdsong-ext.pdf)."  I saw that she also participated in the Kaggle competition on [her Github.](https://github.com/rhine3?tab=repositories)
